dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/TrackerEventsExportControllerTest.java

```diff
@@ -198,10 +194,7 @@ void getEventByIdRelationshipsNoAccessToRelationshipItemTo()
 
         assertFalse( json.isEmpty() );
         JsonArray relationships = json.getArray( "relationships" );
-        assertEquals( 1, relationships.size(),
-            "other endpoints return an empty relationships array, this one currently returns [null]" );
-        assertTrue( relationships.get( 0 ).isNull(),
-            "other endpoints return an empty relationships array, this one currently returns [null]" );
+        assertEquals( 0, relationships.size() );
```

Comment:
I did not see this in the PR to core. For future changes please use https://github.com/dhis2/dhis2-core/blob/fb72648b2f46a97513bb69ce99dcab4757acd74d/dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/utils/Assertions.java#L117 instead. Gives better error messages in case the collection is not empty.